### PR TITLE
Add flag for using sytem Fast-CDR

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

File: cmake/SuperBuild.cmake [hunk 1]
```diff
@@ -49,7 +49,7 @@ if(UAGENT_P2P_PROFILE)
 endif()
 
 # Fast CDR.
-if(NOT UAGENT_USE_SYSTEM_FASTDDS)
+if(NOT UAGENT_USE_SYSTEM_FASTCDR)
```
Comment (Acuadros95) [inline cmake/SuperBuild.cmake:52]:
```suggestion
if(UAGENT_FAST_PROFILE AND NOT UAGENT_USE_SYSTEM_FASTCDR)
```


Is FastCDR part of the FastMiddleware profile? Check:
https://github.com/eProsima/Micro-XRCE-DDS-Agent/blob/3fd419f943cacb35d792fd4558b978a1f57bb3ee/cmake/SuperBuild.cmake#L83-L86

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/446/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/447/)

File: cmake/SuperBuild.cmake (hunk 1 above)
Comment (pablogs9) [reply cmake/SuperBuild.cmake:52]: No, the XRCE-DDS Agent uses FastCDR for message serialization in any case. For example `UAGENT_CED_PROFILE` uses also FastCDR but not FastDDS, so FastCDR is required even if `UAGENT_FAST_PROFILE` is not enabled.

Comment (Acuadros95) [reply cmake/SuperBuild.cmake:52]: Ok then

**Review by Acuadros95 — DISMISSED** [review]

File: colcon.pkg [hunk 2]
```diff
@@ -7,6 +7,7 @@
     ],
     "cmake-args":[
         "-DUAGENT_ISOLATED_INSTALL=OFF",
+        "-UAGENT_USE_SYSTEM_FASTCDR=ON",
```
Comment (pablogs9) [inline colcon.pkg:10]:
```suggestion
        "-DUAGENT_USE_SYSTEM_FASTCDR=ON",
```

**Review by Acuadros95 — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/447/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/448/)

**Pull request merged**